nixos/modules/programs/alvr.nix

```
@@ -0,0 +1,35 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.alvr;
+in
+{
+  options = {
+    programs.alvr = {
+      enable = mkEnableOption (lib.mdDoc "Whether to enable ALVR, the VR desktop streamer.");
```

```suggestion
      enable = mkEnableOption (lib.mdDoc "ALVR, the VR desktop streamer");
```
This is added by `mkEnableOption` already

---

done